lib/metalsmith-browser-sync.js

```diff
@@ -0,0 +1,63 @@
+// Based on metalsmith-browser-sync (Copyright (c) 2015 Mike Dvorscak)
+// https://github.com/mdvorscak/metalsmith-browser-sync
+
+const bs = require('browser-sync').create()
+
+const path = require('path')
+const paths = require('./paths.js') // specify paths to main working directories
+
+const PLUGIN_NAME = 'browser-sync'
+
+const defaultOptions = {
```

Comment:
I think I'd prefer to see the options preserved in **tasks/serve.js**

But not a blocker, can see how we've brought the old plugin back to life